internal/command/permission_checks.go

```diff
@@ -136,3 +138,34 @@ func (c *Commands) checkPermissionDeleteProjectGrantMember(ctx context.Context,
 	}
 	return nil
 }
+
+func (c *Commands) newUserGrantPermissionCheck(ctx context.Context, permission string) UserGrantPermissionCheck {
+	check := c.newPermissionCheck(ctx, permission, project.AggregateType)
+	return func(projectID, projectGrantID string) PermissionCheck {
+		return func(resourceOwner, _ string) error {
+			if projectGrantID != "" {
+				grantErr := check(resourceOwner, projectGrantID)
+				if grantErr != nil {
+					projectErr := check(resourceOwner, projectID)
+					if projectErr != nil {
+						return grantErr
+					}
+				}
```

Comment:
@stebenz we wanted to delete this, right?

Reply:
Yes